src/Common/src/Interop/Linux/procfs/Interop.ProcFsStat.cs

```diff
@@ -67,16 +72,104 @@ internal struct ParsedStat
             //internal long cguest_time;
         }
```

Comment:
Is this struct used anywhere? It doesn't look to be here and is completely commented out